### ISSUE TITLE
remark-lint:awesome-github: False negative when remote name is not `origin`

### ISSUE THREAD

**Issue by hyperupcall** [sindresorhus/awesome-lint#169]
`awesome-lint` fails on the following remote configuration:

```text
$ git remote -v
me      https://github.com/hyperupcall/awesome-mastodon (fetch)
me      https://github.com/hyperupcall/awesome-mastodon (push)
```

This is because the command `git remote get-url --push origin` is ran under the hood. The assumption that `origin` is the remote name does not always hold true. My changes fix that.

## Before

```sh
$ awesome-lint
✖ Linting

  README.md:1:1
  ✖  1:1  Awesome list must reside in a valid git repository  remark-lint:awesome-github

  1 error
```

## After

```text
$ awesome-lint
✔ Linting
```
